src/index.js

```diff
@@ -0,0 +1,42 @@
+import './index.css';
+
+const toDoList = [
+  {
+    description: 'Go swimming',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Visit dentist',
+    completed: false,
+    index: 2,
+  },
+  {
+    description: 'Plan party',
+    completed: false,
+    index: 3,
+  },
+];
+
+const createToDoItemHtml = ({ description }) => {
+  const div = document.createElement('div');
+  div.className = 'to-do-item';
+  div.innerHTML = `
+    <div class="to-do-detail">
+    <input type="checkbox">
+    <h3 class="to-do-dsc">${description}</h3>
+</div>
+<div>
+    <button class="edit-btn"><i class="fa-regular fa-pen-to-square"></i></button>
+    <button class="delete-btn"><i class="fa-solid fa-trash-can"></i></button>
+</div>
+`;
+  return div;
+};
+
+const showToDoItems = () => {
```

Comment:
Please make sure the tasks are rendered in order of the `index` values for each task.
![tasks1](https://user-images.githubusercontent.com/35504149/200022801-99e1f7a4-a1e7-4e9e-bee3-0b3e4bfcf87c.png)

![tasks2](https://user-images.githubusercontent.com/35504149/200022814-2d27d184-e1cb-4848-b001-cbf73a6cb04e.png)